app/javascript/packs/find-out.js

```diff
@@ -93,7 +93,8 @@ const fetchTaggings = (dropdownsObject) => {
       $('.results').empty()
         .removeClass('results--cards-high')
         .addClass('results--cards-high');
-      $('.find-out__header').removeClass('find-out__header--large')
+      $('.find-out__header')
+        .removeClass('find-out__header--large')
```

Comment:
The emoji made me giggle, but I would squash this with the prior commit since they're both relatively minor refactors on the same file